sw/host/opentitanlib/src/test_utils/lc_transition.rs

```diff
@@ -288,8 +308,10 @@ pub fn wait_for_status(jtag: &mut dyn Jtag, timeout: Duration, status: LcCtrlSta
         let polled_status =
             LcCtrlStatus::from_bits(polled_status).context("status has invalid bits set")?;
 
-        // Check for any error bits set.
-        if polled_status.intersects(LcCtrlStatus::ERRORS) {
+        // Check for any error bits set - however, we exclude the status that
+        // we are looking for in this comparison, since otherwise this
+        // function would just bail.
+        if polled_status.intersects(LcCtrlStatus::ERRORS & !status) {
```

Comment:
@timothytrippel FYI - this is needed so that we can wait for specific error statuses.